.evergreen.yml

```
@@ -225,10 +225,16 @@ functions:
     command: shell.exec
     params:
       working_dir: spruce
+      shell: "bash"
```

not sure if bash needs to be in quotes
```suggestion
      shell: bash
```